packages/js/src/utils/requester.ts

```diff
@@ -39,16 +26,11 @@ export const createRequester = (fetchFunction?: typeof fetch) => {
     const response = await (fetchFunction ?? fetch)(...args);
 
     if (!response.ok) {
-      const logtoErrorMessage = await getLogtoErrorParametersByResponse(response);
-      throw new LogtoError(
-        'requester.failed',
-        logtoErrorMessage.error,
-        logtoErrorMessage.error_description,
-        logtoErrorMessage.message
-      );
+      // Expected errors from server
+      throw await getLogtoRequestError(response);
```

Comment:
we can pass `response.json()` to avoid turning `getLogtoRequestError()` into an async function.

Reply:
Refactored, PTAL.